sample-apps/delivery-customizations-sample-app/web/frontend/pages/rename/[id].jsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "@shopify/app-bridge-react";
+import NotFound from "../NotFound";
+
+import { CustomizationForm, CustomizationPageLayout } from "../../components";
+import {
+  useCustomizationForm,
+  useDeliveryCustomization,
+  useUpdateDeliveryCustomization,
+} from "../../hooks";
+
+export default function DeliveryCustomizationDetailPage() {
+  const navigate = useNavigate();
+  const { id } = useParams();
+  const { handleInputChange, setData, data: formData } = useCustomizationForm();
+
+  const { data, isFetching } = useDeliveryCustomization({
+    id,
+  });
+
+  const { mutateAsync: updateCustomization, isLoading } =
+    useUpdateDeliveryCustomization({
+      id,
+    });
+
+  const disabled = isFetching || isLoading;
+
+  const handleSubmit = async () => {
+    if (disabled) return;
+
+    try {
+      await updateCustomization({ payload: formData });
+      navigate("/");
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    if (!data) return;
+
+    const { value: deliveryOptionName, enabled, title, operation } = data;
+
+    setData({
+      deliveryOptionName,
+      enabled,
+      title,
+      operation,
+    });
+  }, [data]);
+
+  const primaryAction = {
+    disabled,
+    onAction: handleSubmit,
+  };
+
+  if (!isFetching && data?.id == null) {
+    return <NotFound />;
+  }
+
+  return (
+    <CustomizationPageLayout
+      title={data?.title || ""}
```

Comment:
Nit: Since this isn't hardcoded, it appears blank until the data loads in. But we know what the type is (and we event [hardcode it on the new page](https://github.com/Shopify/function-examples/pull/120/files#diff-2aea6c08d483bbebfcb092beb1bf6673d6cd98c6c1fa47f01938f64a60744850R43)). I think we should also just hardcode it here too.

Same comment for the other `[id]` pages.